### PR TITLE
Emily

### PR DESCRIPTION
**Pull request by emilyCheroske**

### CODE REVIEW

File: Workouts/HomeView.swift [hunk 1]
```diff
@@ -25,13 +25,15 @@ struct HomeView: View {
...
-                    }
+                    NavigationLink(destination: WorkoutDetail(workout: featuredWorkout!)) {
+                        VStack(alignment: .leading) {
+                            Text("Your latest workout 🏅")
+                                .padding(.vertical)
+                                .font(.system(size: 21, weight: .medium))
+                            FeaturedWorkout(workout: featuredWorkout!)
+                                .padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+                        }
+                    }.padding(.trailing, -22.0) // <-- hide that dang little arrow icon
```
Comment (dawitelias) [inline Workouts/HomeView.swift:36]: Haha nice fix 😛. Hopefully, as SwiftUI evolves, Apple will come up with easy ways to deal with little annoyances like this

File: Workouts/SupportingViews/MapView.swift [hunk 2]
```diff
@@ -29,7 +29,7 @@ struct MapView: UIViewRepresentable {
             // Note * We need to handle workouts/workout types that may not have route data. This myRoute.count > 0 is a safety check - but we should probably handle this in the workout detail page so that we aren't showing the map view at all if we don't have route data
             if let myRoute = route, myRoute.count > 0 {
                 let polyline = MKGeodesicPolyline(coordinates: myRoute.map { return CLLocationCoordinate2D(latitude: $0.coordinate.latitude, longitude: $0.coordinate.longitude) }, count: myRoute.count)
-                uiView.setVisibleMapRect(polyline.boundingMapRect, animated: true)
+                uiView.setVisibleMapRect(polyline.boundingMapRect, animated: false)
```
Comment (dawitelias) [inline Workouts/SupportingViews/MapView.swift:32]: Good call on this 👍

**Review by dawitelias — APPROVED** [review]

File: Workouts/SupportingViews/MapView.swift (hunk 2 above)
Comment (emilyCheroske) [reply Workouts/SupportingViews/MapView.swift:32]: Yea I'm not sure why it animates on scroll but I feel like I don't want to spend a ton of time trying to figure it out right now with other things to do

File: Workouts/HomeView.swift (hunk 1 above)
Comment (emilyCheroske) [reply Workouts/HomeView.swift:36]: Agreeeed, kind of a hack, but it doesn't seem like there is a simple way to say - hey, I don't want an icon 😆

**Pull request merged**